### DISCUSSION TITLE
Any plan to add VALARM option?

### DISCUSSION THREAD

**Discussion by myon1664** [add2cal/add-to-calendar-button#348] [Ideas / Requests]
Hi jekuer,

Is there any plan to add VALARM option for the calendar entries?
This is actually the only option missing for my use cases.

Anyway this library is great! Thanks for sharing it!

Kind regards,
Christian

**Comment by jekuer** [comment]
Actually, no.

2 Reasons:
- it is far away from being somehow reliable. Most calendars do not respect this setting (maybe because of the second reason). Making this an option would create wrong assumptions for the users of the tool, since it would not work in 80-90% of the cases.
- setting the notification should be fully up to the end users and their calendars' default settings. Therefore, trying to force this feature from the outside would intimidate many people. One could even argue that it is against the GDPR rules.

That's why, at the moment, I do not see a valid case for it.
But always open to learn about new use cases and arguments. 🙂
